packages/vscode-extension/src/panels/SidepanelViewProvider.ts

```diff
@@ -33,7 +33,10 @@ export class SidePanelViewProvider implements WebviewViewProvider, Disposable {
     this._view.webview.html = generateWebviewContent(
       this.context,
       this._view.webview,
-      this.context.extensionUri
+      this.context.extensionUri,
+      PREVIEW_WEBVIEW_NAME,
+      PREVIEW_WEBVIEW_PATH,
+      ""
```

Comment:
last parameter is optional so IMO it'd be better to skip it entirely rather than pass empty string